monte_carlo.ipynb

```diff
@@ -39,7 +39,12 @@
     "numberOfCustomerEvents = round(1.1*timeHorizon/meanTimeBetweenCustomerArrival)\n",
     "prices = np.arange(0,15.1,0.1)\n",
     "watchedCompany = 0\n",
-    "numberSimulations = 1000"
+    "numberSimulations = 1000\n",
```

Comment:
Is this now a start configuration that does not process hours to generate the data?

Please also update the information in the header.

Reply:
On top of that, please update the README as we have one notebook less now.

Reply:
It is still the same start configuration as before, it probably shouldn't be approved yet before the names get changed and a good start config is found.